pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/email/SendGridClient.java

```diff
@@ -0,0 +1,125 @@
+package org.broadinstitute.lddp.email;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+import com.google.gson.JsonObject;
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.helpers.mail.objects.Personalization;
+import lombok.NonNull;
+import org.apache.http.HttpStatus;
+import org.broadinstitute.dsm.exception.NotificationSentException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Used for integrating with SendGrid...
+ */
+public class SendGridClient implements EmailClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(SendGridClient.class);
+
+    private static final String LOG_PREFIX = "SendGrid";
+
+    private String sendGridKey;
+    private JsonObject settings = null;
+
+    public void configure(String sendGridKey, JsonObject settings, @NonNull String portalUrl, @NonNull String environment) {
```

Comment:
are `portalUrl` and `environment` used at all?

Reply:
Good catch I forgot to rip these ones out! 👍